dashboards-observability/.cypress/integration/1_event_analytics.spec.js

```diff
@@ -911,3 +912,44 @@ describe('Renders heatmap chart for Chart Style', () => {
     cy.get('stop[stop-color="rgb(214, 191, 87)"]').should('exist');
    });
 });
+
+describe('Renders Data Configurations section for Pie chart', () => {
+  beforeEach(() => {
+    landOnEventVisualizations();
+  });
+
+  it.only('Renders Dimensions and Metrics under Data Configurations for Pie chart', () => {
+    renderPieChart();
+    renderDataConfig();
+  });
+
+  it.only('Validate "Add" and "X" buttons', () => {
+    renderPieChart();
+    cy.get('.euiResizablePanel.euiResizablePanel--middle').contains('Data Configurations');
```

Comment:
`euiResizablePanel.euiResizablePanel--middle` is this name given to component?if yes need to update name with camel case.

Reply:
Its a class name which is already there in the code.